src/merkle_tree/mod.rs

```diff
@@ -133,12 +135,121 @@ impl Pushable for &MerkleTreeTwinProof {
     }
 }
 
+impl MerkleTreeTwinProof {
+    /// Convert a stwo Merkle proof into twin proofs for each pairs of queries.
+    pub fn from_stwo_proof(
```

Comment:
What do you think about documenting parameters always ? They might be self explanatory for you but I dont think it's the case for everyone, and it might be useful to document the parameters so that the Rust doc will be complete.